src/lib.rs

```diff
@@ -1,31 +1,59 @@
-extern crate regex;
-extern crate crypto;
-
-use std::i64;
-use regex::Regex;
-use self::crypto::digest::Digest;
-use self::crypto::sha3::Sha3;
+#![cfg_attr(any(test, bench), feature(test))]
+use crypto::{digest::Digest, sha3::Sha3};
```

Comment:
There's a smaller library for doing this:
```rust
use tiny_keccak::Keccak;

fn keccak256_hash(bytes: &[u8]) -> [u8; 32] {
    let mut keccak = Keccak::v256();
    let mut result = [0u8; 32];
    keccak.update(bytes);
    keccak.finalize(&mut result);
    result
}
```

Reply:
this could also work.
Another thing I want to do is probably use a lazy_static or smth to init the Kecccak hasher when needed and not to create an instance every time